src/domain/provider-service.ts

```diff
@@ -65,11 +67,31 @@ export class ProviderService {
     }
   }
 
-  public async start (providerId: string, serviceType: string) {
-    const service = await this.tequilapiClient.serviceStart({
+  public async getFirstAccessPolicy (): Promise<AccessPolicyDTO | null> {
+    try {
+      const accessPolicy = await this.tequilapiClient.accessPolicies()
+
+      return accessPolicy[0]
```

Comment:
In case empty list is returned, we might throw specific (i.e. 'Empty list of access policies was returned') error instead of `IndexOutOfBox` (or something like that) error.